steering-files/src/main/resources/org/hps/steering/recon/PhysicsRun2019_pass0_recon.lcsim

```diff
@@ -0,0 +1,439 @@
+    <!-- 
+      Steering file for running 2019 Reconstruction on readout data (LCIO or EVIO)
+      created:  9/24/19
+      @author Norman Graf <Norman.Graf@slac.stanford.edu>
+      @author PF <pbutti@slac.stanford.edu>
+    -->
+    <execute>
+        <driver name="EventMarkerDriver"/>
+        <!-- Enable the following if re-processing lcio files -->
+        <!--        <driver name="PreCleanupDriver"/>       -->
+        <!--  Event Reconstruction  -->
+        <!-- Skip events with known bad conditions -->
+        <!-- Not yet defined for 2019 data -->
+        <!--        <driver name="EventFlagFilter"/>  -->
+        <!--RF driver-->
+        <driver name="RfFitter"/>
+        <!-- Ecal reconstruction drivers -->        
+        <driver name="EcalRunningPedestal"/> 
+        <driver name="EcalRawConverter" />
+        <driver name="EcalTimeCorrection"/> 
+        <driver name="ReconClusterer" /> 
+        <driver name="CopyCluster" />
+
+        <!-- Hodoscope drivers -->
+        <driver name="HodoRunningPedestal"/>
+        <driver name="HodoRawConverter"/>
+ 
+        <!-- SVT reconstruction drivers -->
+        <driver name="RawTrackerHitSensorSetup"/>
+        <driver name="RawTrackerHitFitterDriver" />
+        <driver name="TrackerHitDriver"/>
+        
+        <driver name="HelicalTrackHitDriver"/> 
+        
+        <!-- Track finding and fitting using seed tracker. -->
+        
+        <driver name="TrackReconSeed123Conf4Extd56"/>    
+        <driver name="TrackReconSeed123Conf5Extd46"/>
+        
+        <driver name="TrackReconSeed567Conf4Extd123"/>
+        <driver name="TrackReconSeed456Conf3Extd127"/>
+        <driver name="TrackReconSeed356Conf7Extd124"/>
+        <driver name="TrackReconSeed235Conf6Extd147"/> 
+        <driver name="TrackReconSeed234Conf6Extd157"/> 
+
+        <!-- 
+           TrackDataDriver needs to be run before ReconParticleDriver so the
+           ReconstructedParticle types are properly set.
+        -->
+        
+        <driver name="MergeTrackCollections"/>
+        <driver name="GBLRefitterDriver" />
+        <driver name="TrackDataDriver" />
+        <driver name="KalmanPatRecDriver"/>
+        <driver name="ReconParticleDriver" /> 
+        <driver name="ReconParticleDriver_Kalman" /> 
+        
+        <!--  DQM   -->
+              
+        <!-- Following are optional Analysis Drivers  -->
+<!--        
+        <driver name="EcalMonitoring"/>  
+        <driver name="EcalMonitoringCorr"/>  
+        <driver name="SVTMonitoring"/>
+        <driver name="TrackingMonitoring"/>  
+        <driver name="TrackingMonitoringMatched"/>        
+        <driver name="TrackingResiduals"/>
+        <driver name="FinalStateMonitoring"/>          
+        <driver name="V0Monitoring"/>          
+        <driver name="TridentMonitoring"/>           
+        <driver name="MuonCandidateMonitoring"/>    
+        -->      
+             
```

Comment:
Do we need the PreCleanupDriver here so avoid the LCIO file size from becoming something too large?

Reply:
The monitoring drivers are OFF by default.